Had issues with a allocation in a untraceable database so we restarted the to do list api
